changes.md

```diff
@@ -1,3 +1,5 @@
+# 0.2.5
+- Fixing small issues
```

Comment:
Not a useful message. It could just be "Upgraded `nested_assign` to work with Mongoid 7.x